src/pages/Poll.jsx

```diff
@@ -75,7 +88,20 @@ function Poll(props) {
           )}
         </Formik>
       ) : (
-        <div>User</div>
+        <Formik
+          initialValues={questions}
+          onSubmit={values => console.log(values)}
```

Comment:
![image](https://user-images.githubusercontent.com/39658211/80487153-a36c4600-895c-11ea-977a-fb19fa6eac32.png)
Niestety nie może być coś takiego bo na backendzie nie beda wiedzieli co z tym zrobić. Będziesz musiał dopisać funkcję w ciele komponentu `formatAnswers()`  która bedzie przyjmować `values` oraz zwracać obiekt z tablicą odpowiedzi (obiektów) w postaci id pytania i odpowiedzi oraz podpisem o ile takowy istnieje. Jak jest to odpowiedz wielokrotnego wyboru wartosci znajdują się w tablicy:

```Javascript
{ 
  answers: [
  {
    id: "689ef100-d5d5-4ed4-80da-d18165ae1003",
    value: "Alpaka"
  },
  {
     id: "f9674ec9-561d-469b-9a4d-94837ff81acc"
     value: ["Owca", "Lama"]
  },
  {
    id: "c67006dd-9606-42a9-bbea-b6074c93120b",
    value: "Odpowiedź na pytanie trzecie"
  }
],
signature: "Moj Podpis"
}
```

Dlaczego tak? Otóż twórca ankiety może edytować pytania i indeksy tablicy mogą się zmieniać a mając ID pytania ogarną sobie typ pytania.

```suggestion
          onSubmit={values => {
            const formattedAnswers = formatAnswers(values);
            console.log(formattedAnswers);
          }}
```